crates/wit-parser/src/resolve.rs

```diff
@@ -1318,6 +1318,21 @@ impl Remap {
     ) -> Result<()> {
         let include_world_id = self.worlds[include_world.index()];
         let include_world = &resolve.worlds[include_world_id];
+        let mut names_ = names.clone().to_owned();
+
+        // remove all imports and exports that match the names we're including
+        for import in include_world.imports.iter() {
+            self.remove_matching_name(import, &mut names_);
+        }
+        for export in include_world.exports.iter() {
+            self.remove_matching_name(export, &mut names_);
+        }
+        if !names_.is_empty() {
+            bail!(Error {
+                msg: format!("no import or export kebab-name `{}`. Note that an ID does not support renaming", names_[0].name),
```

Comment:
Just a note that there might be some confusion about kebab-name and ID. Would saying "interface id" be clearer?

There is potentially a larger conversation about naming here as well: whether "kebab-name" is an appropriate label considering some ids also use kebab case.

Reply:
The component model repo is probably a better place for this question. I used the terminology from the WIT description

> In the component model imports to a component either use an ID or a kebab-name, and in WIT this is reflected in the syntax:

https://github.com/WebAssembly/component-model/blob/main/design/mvp/WIT.md#wit-worlds